### PR TITLE
HDDS-5848. Introduce more replication metrics

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Capture few more metrics around replication.

https://issues.apache.org/jira/browse/HDDS-5848

## How was this patch tested?

Updated unit test.

https://github.com/adoroszlai/hadoop-ozone/actions/runs/1368400919

### CODE REVIEW

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/MeasuredReplicator.java [hunk 1]
```diff
@@ -67,12 +73,15 @@ public void replicate(ReplicationTask task) {
         (Instant.now().getNano() - task.getQueued().getNano()) / 1_000_000;
     queueTime.incr(msInQueue);
     delegate.replicate(task);
+    long end = System.currentTimeMillis();
     if (task.getStatus() == Status.FAILED) {
       failure.incr();
+      failureBytes.incr(task.getTransferredBytes());
+      failureTime.incr(end - start);
```
[the remark this answers is not in the document]
Comment (adoroszlai) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/MeasuredReplicator.java]: Thanks @ayushtkn for the idea.  I have updated the patch.

**Comment by adoroszlai** [comment]
@kerneltime Can you please take a look, too?

**Review by ayushtkn — APPROVED** [review]
Thanx @adoroszlai, Changes looks good to me.
Can wait for sometime for @kerneltime  to have a look as well.

**Comment by adoroszlai** [comment]
Thanks @ayushtkn and @guihecheng for the review.

**Pull request merged**